### ISSUE TITLE
Squiz VariableCommentSniff doesn't always detect a missing comment

### ISSUE THREAD

**Issue by aik099** [squizlabs/PHP_CodeSniffer#1014]
The `Squiz.Commenting.VariableComment` sniff reports `WrongStyle` error on the class property without DocBlock. I think, that it should check if comment is present at all before validating it as `FunctionComment` sniff does in some other standard (PSR2 if I'm not mistaken).

**Comment by gsherwood** [comment]
The error I get is `Missing member variable doc comment (Squiz.Commenting.VariableComment.Missing)` which means the sniff has found an inline comment and has assumed that is the docblock because it is above the variable.

You'd need to post sample code to be clearer, but this sniff does report missing comments.

**Comment by aik099** [comment]
Here is code example:

```php
<?php
/*
 * The MIT License
 */
namespace chobie\Jira;


class IssueType
{

	protected $self;

}
```

**Comment by gsherwood** [comment]
Fixed now. Thanks for the code example.

**Comment by aik099** [comment]
Thanks.
